cola/widgets/stash.py

```diff
@@ -206,21 +230,14 @@ def stash_save(self):
             parent=self)
         if not ok or not stash_name:
             return
-        # Sanitize the stash name
-        stash_name = utils.sanitize(stash_name)
-        if stash_name in self.names:
-            Interaction.critical(
-                N_('Error: Stash exists'),
-                N_('A stash named "%s" already exists') % stash_name)
-            return
```

Comment:
Nicely done.  True, the stash "names" are really just the messages, and don't need to be unique.